build-utils/src/index.ts

```diff
@@ -32,27 +34,30 @@ export async function build(projectDir: string, options?: Partial<BuildOptions>)
   const {
     skipNpmInstall,
     logger,
+    quiet,
   } = {
     ...DEFAULT_OPTIONS,
     ...options,
   };
   logger.log('Building and bundling your app! This may take a few moments, please wait');
   const stagingDir = await mkdtemp(pathResolve(tmpdir(), 'reshuffle-bundle-'), { encoding: 'utf8' });
 
+  const stdioInherit = [0, 1, 2];
+  const stdioSuppress = [0, 2, 2];
```

Comment:
Note that this is not exactly `suppress` (which I would assume is `redirect to /dev/null`, but `redirectToStderr`).

Both are probably useful in some cases, but the naming should be consistent (since this is not exactly `quiet`).

Perhaps our command line can have an additional `--quiet` flag which suppresses output, and does not even print it to stderr.